kolibri/plugins/coach/assets/src/views/common/QuizActive.vue

```diff
@@ -37,8 +37,8 @@
     computed: {
       label() {
         return this.active
-          ? this.coachCommon$tr('quizActiveLabel')
-          : this.coachCommon$tr('quizInactiveLabel');
```

Comment:
This might run into the noun-adjective agreement issue (e.g. "active [quiz]" and "active [lesson"] would be translated differently.), so I would say to keep 'quizActiveLabel' and 'lessonActiveLabel' separate in coach strings.

Reply:
@jonboiser Do you know offhand if/where there are other contexts in which the word "Active" would refer to an elided "quiz"? 

If this is the only component, then I will define the string on that component directly with "activeQuizLabel" and "inactiveQuizLabel" rather than using the common "(in)activeLabel" strings.

Reply:
> I will define the string on that component directly 

that seems like a safe bet for cases like this one in general